Document about excluding hosts (#1453)
theforeman/foreman-documentation: guides/common/modules/proc_using-red-hat-insights.adoc

If the host has recommendations listed, click on the number of recommendations.
On the *Insights* tab, the vertical ellipsis next to the *Remediate* button provide a *Go To {Project} Insights page* link to information for the system, and a *View in {Team} Insights* link to host details on the console.

.Excluding hosts from rh-cloud and insights-client reports

> Starting from {Project} {ProductVersion}, you can omit host reporting from _rh-cloud_ and _insight-client_ reports by setting the `host_registration_insights` parameter to *False*.

Reviewer: ```suggestion
You can set the `host_registration_insights` parameter to *False* to omit host reporting from _rh-cloud_ and _insight-client_ reports.
```

We generally don't say "starting with version x" because users will always get the docs that match their Foreman/Katello/Satellite/orcharhino version.

{Project} will exclude the hosts from _rh-cloud_ reports and block _insight-client_ from uploading a report to the cloud.

> This parameter can also be set at the organization, hostgroup, subnet, and domain level.

Reviewer: Please make it "active", i.e. "you can set" instead of "can be set".

It automatically prevents new hosts from being uploaded as long as they are associated with the entity.

> In the case where the host is already reported while setting the parameter to false, it can still be removed by the aging process from the cloud.

Reviewer: ```suggestion
If a host already sends reports to {Project} and you set the parameter to false, it can still be removed by the aging process from the cloud.
```

I don't understand the last part of the sentence. Which cloud? Where? What's the "aging process"?

Author: Addressed.
